libsolidity/analysis/ControlFlowAnalyzer.cpp

```diff
@@ -141,7 +142,7 @@ void ControlFlowAnalyzer::checkUninitializedAccess(CFGNode const* _entry, CFGNod
 			exitInfo.uninitializedVariableAccesses.begin(),
 			exitInfo.uninitializedVariableAccesses.end()
 		);
-		boost::range::sort(
+		ranges::sort(
```

Comment:
@cameel @ekpyron I don't know who said it, or if I'm remembering correctly, but didn't someone say these are not equivalent?

Reply:
I only remember something like that about `boost::remove_erase_if()` vs `ranges::actions::remove_if()`? (https://github.com/ethereum/solidity/pull/11311#issuecomment-826132661).

I think that we do not have this problem with `sort()`. Both sort in place.